README.md

```diff
@@ -9,3 +9,7 @@ For the Java implementation, see the `java` directory.
 For the Python implementation, see the `python` directory.
 
 For the Node.js implementation, see the `nodejs` directory.
+
+For the C# implementation, see the `dotnet` directory.
+
+For the Go implementation, see the `gocse` directory.
```

Comment:
Nitpick: this directory name doesn't follow the same naming scheme so I'd probably change it to either "go" or "golang" to match. If those don't work for whatever reason, maybe "gocsfle" would be easier to understand.

Reply:
I had chosen 'cse' because in order to use csfle you have to use the build tag `cse`. I think I had avoided using `go` because it could be confused with the system `go`, but I'll pay around with it.